vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -10930,6 +10963,22 @@ unit:KiloGM-PER-HR
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Kilogram per Hour"@en ;
 .
+unit:KiloGM-PER-J
+  a qudt:DerivedUnit ;
+  a qudt:Unit ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.001 ;
+  qudt:definedUnitOfSystem sou:SI ;
+  qudt:hasDimensionVector qkdv:A0E0L-2I0M1H0T2D0 ;
```

Comment:
Whoops! Should be qkdv:A0E0L-2I0M0H0T2D0

Reply:
Thanks @steveraysteveray. I appreciate your patience on this PR